Library/Homebrew/dev-cmd/verify.rb

```diff
@@ -0,0 +1,87 @@
+# typed: strict
+# frozen_string_literal: true
+
+require "abstract_command"
+require "formula"
+require "date"
+require "attestation"
+
+module Homebrew
+  module DevCmd
+    class VerifyCmd < AbstractCommand
```

Comment:
We only add the `Cmd` suffix when needed to avoid namespace collisions:
```suggestion
    class Verify < AbstractCommand
```
(This will require re-running `brew tc --update`)